src/cpu/aarch64/matmul/acl_lowp_matmul.hpp

```diff
@@ -0,0 +1,268 @@
+    }
+    acl_obj.dst_tensor.allocator()->init(almc.dst_tensor_info);
+
+    acl_obj.gemm.configure(&acl_obj.src_tensor, &acl_obj.wei_tensor,
+            almc.with_bias ? &acl_obj.bia_tensor : nullptr,
+            &acl_obj.dst_tensor);
+
+    return status::success;
+}
+
+struct acl_lowp_matmul_resource_t : public resource_t {
+    acl_lowp_matmul_resource_t()
+        : acl_obj_(utils::make_unique<acl_lowp_matmul_obj_t>()) {}
+
+    status_t configure(const acl_lowp_matmul_conf_t &almc) {
+
+        if (!acl_obj_) return status::out_of_memory;
+
+        acl_obj_->src_tensor.allocator()->init(almc.src_tensor_info);
+        acl_obj_->wei_tensor.allocator()->init(almc.wei_tensor_info);
+        if (almc.with_bias) {
+            acl_obj_->bia_tensor.allocator()->init(almc.bia_tensor_info);
+        }
+        acl_obj_->dst_tensor.allocator()->init(almc.dst_tensor_info);
+
+        acl_obj_->gemm.configure(&acl_obj_->src_tensor, &acl_obj_->wei_tensor,
+                almc.with_bias ? &acl_obj_->bia_tensor : nullptr,
+                &acl_obj_->dst_tensor);
+
+        return status::success;
+    }
+
+    acl_lowp_matmul_obj_t &get_acl_obj() const { return *acl_obj_; }
+
+    DNNL_DISALLOW_COPY_AND_ASSIGN(acl_lowp_matmul_resource_t);
+
+private:
+    std::unique_ptr<acl_lowp_matmul_obj_t> acl_obj_;
+};
+
+struct acl_lowp_matmul_t : public primitive_t {
+    struct pd_t : public dnnl::impl::cpu::matmul::cpu_matmul_pd_t {
+
+        pd_t(const matmul_desc_t *adesc, const primitive_attr_t *attr,
+                const cpu_matmul_pd_t *hint_fwd_pd)
+            : cpu_matmul_pd_t(adesc, attr, hint_fwd_pd), almc_() {}
+
+        using cpu_matmul_pd_t::cpu_matmul_pd_t;
+
+        DECLARE_COMMON_PD_T(
+                "lowp_gemm:acl", acl_lowp_matmul_t, USE_GLOBAL_SCRATCHPAD);
+
+        status_t init(engine_t *engine) {
+
+            VDISPATCH_MATMUL(
+                    set_default_formats(), "failed to set default formats");
+            using smask_t = primitive_attr_t::skip_mask_t;
+            VDISPATCH_MATMUL(attr()->has_default_values(smask_t::scales_runtime
+                                     | smask_t::zero_points_runtime),
+                    "only scale and zero point attrs supported");
+            VDISPATCH_MATMUL(attr()->scales_.get(DNNL_ARG_SRC).mask_ == 0
+                            && attr()->zero_points_.get(DNNL_ARG_SRC) == 0
+                            && attr()->scales_.get(DNNL_ARG_WEIGHTS).mask_ == 0
+                            && attr()->zero_points_.get(DNNL_ARG_WEIGHTS) == 0,
+                    "Common scales and zero points only");
+
+            VDISPATCH_MATMUL(!has_runtime_dims_or_strides(),
+                    VERBOSE_RUNTIMEDIM_UNSUPPORTED);
+            VDISPATCH_MATMUL(attr()->scales_.has_default_values(
+                                     {DNNL_ARG_SRC, DNNL_ARG_WEIGHTS}),
+                    "only src and weights scales are supported");
+
```

Comment:
VDISPATCH_MATMUL(attr()->zero_points_.has_default_values(
                                     {DNNL_ARG_SRC, DNNL_ARG_WEIGHTS}),
                    "only src and weights zero points are supported");